libmambapy/src/libmambapy/bindings/legacy.cpp

```diff
@@ -526,25 +554,123 @@ bind_submodule_impl(pybind11::module_ m)
         .def_static("whoneeds", &Query::whoneeds)
         .def_static("depends", &Query::depends);
 
-    py::class_<SubdirData>(m, "SubdirData")
```

Comment:
How should we handle the removal of this API?

Reply:
It's handle via `SubdirDataMigrator` which keep it as it is in Python, but uses a wrapped class to avoid binding the same class twice in Pybind.